apps/generator/docs/deprecate-cli.sj-ag.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Deprecate Cli.js"
```

Comment:
as in my other comment, it is not a source file that we deprecate but the AsyncAPI Generator CLI that has different technical name. See how others still promote it in template readme files -> https://github.com/asyncapi/java-spring-cloud-stream-template?tab=readme-ov-file#how-to-use-this-template as `ag`